webapp/components/admin_console/localization_settings.jsx

```diff
@@ -49,23 +49,16 @@ export default class LocalizationSettings extends AdminSettings {
         return (
             <h3>
                 <FormattedMessage
-                    id='admin.general.title'
-                    defaultMessage='General Settings'
+                    id='admin.general.localization'
```

Comment:
It's not actually renamed, I just used a different but existing translation. I didn't delete the old one though so doing that..